app/views/offices/index.html.erb

```diff
@@ -0,0 +1,75 @@
+<%- @page_title = 'Locator Maps' -%>
+
+<div class="usa-grid usa-section usa-layout-docs">
+  <%= render 'shared/sidenav' %>
+
+  <div class="usa-width-three-fourths usa-layout-docs-main_content">
+    <h1><%= @page_title %></h1>
+
+    <p>Use the search form below to find local Transportation Offices (<%= abbr_tag('pppo') %>s) and Truck Weight Scales.</p>
+
+    <div id="geolocation-notice" hidden>
+      <p>You may also search for Transportation Offices and Truck Weight Scales using your device’s geolocation features. To use these features, you will be prompted to allow access to your location. We use the information provided by your device for the purposes of finding nearby Transportation Offices and Truck Weight Scales but <b>we do not store your location</b>.</p>
+    </div>
+
+    <div id="error-container">
+      <%- if @error_message -%>
+        <div class="usa-alert usa-alert-error" role="alert">
+          <div class="usa-alert-body">
+            <p class="usa-alert-text"><%= @error_message %></p>
+          </div>
+        </div>
+      <%- end -%>
+    </div>
+
+    <div class="search-forms-container">
+      <%= form_with url: offices_path, class: 'usa-form zip-code-search-form', id: 'search-code-zip-form', method: 'get' do |form| %>
+        <%= form.label :postal_code, 'Search by ZIP Code', class: 'usa-input-required usa-sr-only' %>
+        <%= form.number_field :postal_code, aria: { required: true }, id: 'postal_code', pattern: '^\d{5}$', placeholder: 'e.g. 20500', require: true, value: params[:postal_code] %>
+
+        <button>Search by ZIP Code</button>
+      <% end %>
+
+      <%= form_with url: offices_path, class: 'usa-form coordinates-search-form', html: { hidden: true }, id: 'coordinates-search-form', method: 'get' do |form| %>
+        <%= form.hidden_field :coordinates, id: 'coordinates' %>
+        <button>Search by Your Location</button>
```

Comment:
What about "Search by <i>My</i> Location" instead of <i>Your</i> Location?

Reply:
We're mixing perspective throughout the UI and… I dunno which we should standardize on.

@lfantano @LillyMadigan Do y'all have an opinion on the perspective we use? Like… should the website be having a conversation with the user ("Search by Your Location") or should the website be an extensive of the user ("Search by My Location")…?

Or am I overthinking and the shifting perspectives doesn't matter? 😂

Reply:
@jgarber623-gov I prefer that the content and style is consistent. Are there other instances of Your vs My? You are not over thinking this!! Language choices set the tone for the experience the customer is going to have. "My" can appear to give the user power in the process. "Your" suggests something that's been tailored for the user. @lfantano do you have a preference?